client/home/lib/myteam/components/hometeamsendinvites/view.coffee

```diff
@@ -2,10 +2,17 @@ kd = require 'kd'
 React = require 'app/react'
 List = require 'app/components/list'
 CheckBox = require 'app/components/common/checkbox'
+isEmailValid = require 'app/util/isEmailValid'
 
 
 module.exports = class HomeTeamSendInvitesView extends React.Component
 
+  constructor: (props) ->
+
+    super props
+
+    @state = { filledInputCount : {} }
```

Comment:
pls rename the key, it's not holding a decimal anymore so you can't name it as `...Count`